kong.conf.default

```diff
@@ -547,6 +547,12 @@
                          # to Kong administrators *only*. This value accepts
                          # IPv4, IPv6, and hostnames.
                          #
+                         # It is highly recommended to avoid binding Kong to all of your
+                         # interfaces, by using values such as 0.0.0.0:8001
```

Comment:
```suggestion
                         # It is highly recommended to avoid exposing the Admin API to public                         # interface(s), by using values such as 0.0.0.0:8001
```

Reply:
Ping @kikito ^^

Reply:
ping @kikito